src/.editorconfig

```diff
@@ -1,7 +1,6 @@
 ################################################################
 # Global preferences                                           #
 ################################################################
-root = true
```

Comment:
We'll need to put this in the actual root `.editorconfig` too

Reply:
Sorry, thought it was already was and didn't check. More haste less speed >.<